corehq/tests/app_test_db_dependencies.yml

```diff
@@ -157,15 +151,9 @@ corehq.apps.receiverwrapper:
   - django_prbac
   - phonelog
   - tastypie
-  - touchforms.formplayer # required by user.delete()
-  # unfortunately domain.delete() needs all of these - otherwise they could be removed
-  - casexml.apps.stock
-  - corehq.apps.accounting
-  - corehq.apps.commtrack
-  - corehq.apps.smsforms
-  - custom.logistics
-  - custom.ilsgateway
-  - custom.ewsghana
```

Comment:
I didn't go through all the apps and remove the unnecessary dependencies, just these 2.